core/src/test/java/org/apache/iceberg/jdbc/TestJdbcCatalog.java

```diff
@@ -140,7 +140,6 @@ private JdbcCatalog initCatalog(String catalogName, Map<String, String> props) {
 
     properties.put(JdbcCatalog.PROPERTY_PREFIX + "username", "user");
     properties.put(JdbcCatalog.PROPERTY_PREFIX + "password", "password");
-    properties.put(JdbcUtil.SCHEMA_VERSION_PROPERTY, JdbcUtil.SchemaVersion.V1.name());
```

Comment:
for other reviewers: this was changed so that the default tests all run with schem V0 (not view-aware). We then decided to add a separate test class that runs all catalog tests with schema V1